docs/src/modules/components/overview/Customization.tsx

```diff
@@ -0,0 +1,389 @@
+  };
+  const handleChangeLayout = (_event: React.MouseEvent<HTMLElement>, newLayout: Layout) => {
+    setStyleConfig((prev) => ({ ...prev, layout: newLayout }));
+  };
+  const handleChangeCorner = (_event: React.MouseEvent<HTMLElement>, newCorner: Corner) => {
+    setStyleConfig((prev) => ({ ...prev, corner: newCorner }));
+  };
+  const handleChangeDensity = (_event: React.MouseEvent<HTMLElement>, newDensity: Density) => {
+    setStyleConfig((prev) => ({ ...prev, density: newDensity }));
+  };
+
+  const handleChangeTypography = (
+    _event: React.MouseEvent<HTMLElement>,
+    newTypography: TypographyType,
+  ) => {
+    setStyleConfig((prev) => ({ ...prev, typography: newTypography }));
+  };
+
+  return (
+    <React.Fragment>
+      <Divider />
+      <Stack spacing={4} py={8} alignItems="center">
+        <Stack spacing={1} sx={{ maxWidth: { xs: '500px', md: '100%' } }}>
+          <Typography variant="body2" color="primary" fontWeight="semiBold" textAlign="center">
+            Customization
+          </Typography>
+          <Typography
+            variant="h4"
+            component="h2"
+            fontWeight="semiBold"
+            color="text.primary"
+            textAlign="center"
+          >
+            Highly customizable components
+          </Typography>
+          <Typography variant="body1" color="text.secondary" textAlign="center">
+            Easily adaptable to any style, our components leverage Material Design for
+            out-of-the-box elegance and support extensive customization to perfectly align with your
+            branding.
+          </Typography>
+
+          <Stack
+            maxWidth="xl"
+            direction={{ xs: 'column', md: 'row' }}
+            sx={{
+              display: 'flex',
+              border: '1px solid',
+              borderColor: 'divider',
+              borderRadius: 1,
+            }}
+          >
+            <Box
+              component="div"
+              sx={(theme) => ({
+                display: 'flex',
+                borderRadius: {
+                  xs: `0 0 ${theme.shape.borderRadius}px ${theme.shape.borderRadius}px`,
+                  md: `${theme.shape.borderRadius}px 0 0 ${theme.shape.borderRadius}px`,
+                },
+
+                flexBasis: '70%',
+                backgroundImage: `radial-gradient(${theme.palette.divider} 0.8px, ${theme.palette.background.paper} 0.8px)`,
+                backgroundSize: '10px 10px',
+                order: { xs: 2, md: 1 },
+                alignItems: 'center',
+                justifyContent: 'center',
+              })}
+            >
+              <LocalizationProvider dateAdapter={AdapterDayjs}>
+                <CustomTheme
+                  selectedTheme={styleConfig.selectedTheme}
+                  config={styleConfig}
+                  mode={brandingTheme.palette.mode}
+                />
+              </LocalizationProvider>
+            </Box>
+            <Box
```

Comment:
What do you think about making this section horizontally scrollable on small screen sizes? Currently it's hard to see the changes you're making from clicking the toggle buttons due to most of the screen being taken up by the options.

Something like: 

<img src="https://github.com/user-attachments/assets/a9fb3043-cff4-46b6-a14a-5a0dae2da6bf" width="320" />